lldb/include/lldb/Core/Telemetry.h

```diff
@@ -146,6 +155,59 @@ struct DebuggerInfo : public LLDBBaseTelemetryInfo {
   void serialize(llvm::telemetry::Serializer &serializer) const override;
 };
 
+struct ExecModuleInfo : public LLDBBaseTelemetryInfo {
+  lldb::ModuleSP exec_mod;
```

Comment:
Let's drop the `exec_` prefix as this is relatively obvious from the class it belongs to and we don't consistently prefix all the members. 
```suggestion
  lldb::ModuleSP exec_mod;
```